crates/examples/infra/mod.rs

```diff
@@ -560,18 +557,19 @@ pub trait RunDA<
         let consensus_lock = context.hotshot.get_consensus();
         let consensus = consensus_lock.read().await;
         let total_num_views = usize::try_from(consensus.locked_view.get_u64()).unwrap();
-        // When posting to the orchestrator, note that the total number of views also include un-finalized views.
+        let failed_num_views = total_num_views - num_successful_commits; // could include uncommitted views
```

Comment:
Could we change this from an end comment to an above comment? It can make the lines really long.